### PR TITLE
report: rework dockerfile and report script to be more general

### PR DESCRIPTION
**Pull request by iand**

### CODE REVIEW

**Comment by iand** [comment]
None of the github code scan alerts appear to be correct - I'm not logging the password

File: report/lib/lib_db.py [hunk 1]
```diff
@@ -410,26 +411,29 @@ def get_new_agent_versions(self) -> pd.DataFrame:  # DONE
...
-        print("Getting new protocols in the specified time interval...")
-        cur = self.conn.cursor()
-        cur.execute(
-            f"""
-            SELECT EXTRACT('epoch' FROM p.created_at), p.protocol
-            FROM protocols p
-            WHERE created_at > {self.start}
-              AND created_at < {self.end}
-            ORDER BY p.created_at
+        try:
```
Comment (dennis-tra) [inline report/lib/lib_db.py:414]: I guess this try/catch thing can be removed now?

File: report/main.py [hunk 2]
```diff
@@ -126,71 +163,83 @@ def generate_ipfs_report():
...
-        top_rotating_nodes=top_rotating_nodes,
-        ip_address_count=fmt_thousands(ip_address_count),
-        # top_updating_peers=top_updating_peers,
-    )
-
-    with open(f"report-{calendar_week}.md", "w") as f:
-        f.write(outputText)
+
+    print("Rendering template...")
+    try:
```
Comment (dennis-tra) [inline report/main.py:214]: Can this try/catch be removed here and next to `open(...)`?

**Review by dennis-tra — APPROVED** [review]

**Comment by dennis-tra** [comment]
LGTM 👍

**Comment by iand** [comment]
@dennis-tra can you merge please? I don't have permission

**Pull request merged**